thunder/dynamo/compiler.py

```diff
@@ -1,7 +1,25 @@
+from enum import Enum, auto
+import dataclasses
+from typing import List, Dict, Optional, Tuple
+from collections.abc import Callable
+import pprint
+import itertools
+import copy
+from functools import partial
+import operator
+import inspect
+
 import torch
+from torch.fx.passes.split_module import split_module as fx_split_module
```

Comment:
I recommend avoiding renaming imported functions. I tried searching for code in the PyTorch repo "fx_split_module" and there were no results only after that did I notice this import rename.
```suggestion
from torch.fx.passes.split_module import split_module
```